internal/internal_workflow_test.go

```diff
@@ -1308,3 +1308,39 @@ func (t *tracingInterceptor) ExecuteWorkflow(ctx Context, workflowType string, a
 	t.trace = append(t.trace, "ExecuteWorkflow "+workflowType+" end")
 	return result
 }
+
+type WorkflowOptionTest struct {
+	suite.Suite
+}
+
+func TestWorkflowOption(t *testing.T) {
+	suite.Run(t, new(WorkflowOptionTest))
+}
+
+func (t *WorkflowOptionTest) TestKnowQueryType_NoHandlers() {
+	wo := workflowOptions{queryHandlers: make(map[string]func([]byte) ([]byte, error))}
```

Comment:
nit: if this would be like
```
wo := workflowOptions{
  queryHandlers: map[string]func([]byte) ([]byte, error){},
}
```

this would be more obvious it is empty.
+ consider renaming to TestKnowQueryType_No**User**Handlers or something like this. Then it is clear there is no other handlers other than default.

Reply:
I think both are good to read.
Though, Uber style guide is against your proposed approach. https://github.com/uber-go/guide/blob/master/style.md#initializing-maps